apps/src/templates/instructions/TopInstructionsCSF.jsx

```diff
@@ -148,9 +149,17 @@ const styles = {
     width: 'calc(100% - 20px)',
     float: 'left'
   },
+  containedLevelBackground: {
+    backgroundColor: color.background_gray,
+    overflowY: 'scroll',
+  },
   containedLevel: {
-    padding: CONTAINED_LEVEL_PADDING,
-    backgroundColor: '#f2f2f2',
+    paddingTop: CONTAINED_LEVEL_PADDING,
+    paddingLeft: CONTAINED_LEVEL_PADDING,
+    paddingRight: CONTAINED_LEVEL_PADDING,
+  },
```

Comment:
nit: might be more extensible to put these into a `containedLevelStyles` object, a la the `craftStyles`